modules/pricer/src/main/java/com/opengamma/strata/pricer/fx/BlackFxVanillaOptionProductPricer.java

```diff
@@ -74,7 +77,7 @@ public double price(
 
```

Comment:
Improve price javadoc.
The price of the product is the value on the valuation date for one unit of the base currency and is expressed in the counter currency. The price does not take into account the long/short flag. See {@link #presentValue} for scaling and currency.